### ISSUE TITLE
tio.LabelTransform not present

### ISSUE THREAD

**Issue by dmus** [fepegar/torchio#755]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Problem summary

See the code, it is not a bug, but it feels unintuitive to use `tio.transforms.preprocessing.label.label_transform.LabelTransform` for a custom LabelTransform and `tio.IntensityTransform` for an IntensityTransform

### Code for reproduction

```python
import torchio as tio

from rndeep.Utils.subject import Subject


class MyLabelTransform(tio.transforms.preprocessing.label.label_transform.LabelTransform):
    
    def apply_transform(self, subject: Subject) -> Subject:
        for labelmap in self.get_images(subject):
            # do something with labelmap
            pass
        return subject


class MyIntensityTransform(tio.IntensityTransform):

    def apply_transform(self, subject: Subject) -> Subject:
        for labelmap in self.get_images(subject):
            # do something with labelmap
            pass
        return subject
    
intensitytransform = MyIntensityTransform()
labeltransform = MyLabelTransform
```

### Actual outcome

it works

### Error messages

```python-traceback
no errors
```

### Expected outcome

I expect that I could also use tio.LabelTransform

### System info

```shell
Platform:   Linux-5.4.0-62-generic-x86_64-with-Ubuntu-18.04-bionic
TorchIO:    0.18.68
PyTorch:    1.10.0+cu102
SimpleITK:  1.2.4 (ITK 4.13)
NumPy:      1.19.5
Python:     3.6.9 (default, Jan 26 2021, 15:33:00) 
[GCC 8.4.0]
```

**Comment by fepegar** [comment]
Thanks, @dmus. It will be available in the next release.
